src/Model/Volunteer.hs

```diff
@@ -0,0 +1,21 @@
+module Model.Volunteer where
+
+import Import
+
+import Control.Monad.Trans.Writer.Strict (tell)
+
+fetchAllInterestsDB :: DB [Entity Interest]
+fetchAllInterestsDB =
+    select $
+    from $ \i ->
+    pure i
```

Comment:
This function can, and should, be replaced with `selectList [] []` (which should be inlined).